src/OSInfo.cpp

```diff
@@ -23,6 +23,20 @@
 #include "Version.h"
 
 #if defined(Q_OS_WIN)
+// regString converts a wchar_t string of size to a
+// QString. If the string contains a NUL value, that
+// NUL value will terminate the string.
+static QString regString(wchar_t *string, int size) {
+	if (size <= 0) {
```

Comment:
Well. -1 is documented for the qt function but the rest of the negative range isn't. I don't expect it to do anything good ;) But that gets it input from wcsnlen which doesn't take a signed element count so no behavior to document there in the first place ;)